app/controllers/users/two_factor_authentication_controller.rb

```diff
@@ -7,8 +7,10 @@ class TwoFactorAuthenticationController < ApplicationController
     def show
       if current_user.totp_enabled?
         redirect_to login_two_factor_authenticator_url
-      elsif current_user.two_factor_enabled?
+      elsif current_user.phone_enabled?
         validate_otp_delivery_preference_and_send_code
+      elsif current_user.piv_cac_enabled?
```

Comment:
Should the last 2 conditions be reversed, so that if someone has both PIV/CAC and phone, it defaults to PIV/CAC?

Reply:
I was going for minimal touch on this. If doing so doesn't result in a mass of tests breaking, I don't see why I wouldn't. Let me see what happens.